docs/Getting Started/Install and Run/index.md

```diff
@@ -126,4 +127,68 @@ If Core was built with `EXTENDED_POLICY=EXTERNAL_PROPRIETARY`, the `core.sh` scr
 ```bash
 ./core.sh <command> false
 ``` 
-!!!
+## A cMake Exercise
+
+To Build Core using the HTTP Feature Support Option, do the following:
```

Comment:
Probably best not to use HTTP as the example, as it is missing a lot of features that would normally be available for policies. Ideally we would want to phase out anything besides EXTERNAL_PROPRIETARY at some point, but for now this is just a legacy mode we support. Probably best to use EXTERNAL_PROPRIETARY as the example here

Reply:
OK